doc/reference/make.md

```diff
@@ -9,36 +9,184 @@ See also the [guide](/guide/build.md).
 
 ## Overview
 
-Please write me!
+By convention, Gerbil projects are built by invoking a `./build.ss` script,
+typically starting with `#!/usr/bin/env gxi` that defines a function `main`
+that will build the project when called with an empty list.
+The libraries documented below are meant to be used in such a script,
+to do most of the work in building your project.
+Note that if you're using low-level Gambit functions or FFI, you may want to use
```

Comment:
that's for syntax dependencies on `_gambc#` macros only, we should be more explicit about it.

Reply:
Better?

"Note that if your project has syntax dependencies on low-level Gambit code in gambit/lib/_gambit#.scm, you may want to use #!/usr/bin/env gxi-build-script at which point you also need to explicitly call (main) in your script."